### ISSUE TITLE
comparison alternatives

### ISSUE THREAD

**Issue by Konfekt** [habamax/vim-minisnip#1]
Dear @habamax, would you mind quickly comparing your plug-in to the similar alternatives
[vim-minisnip](https://github.com/tckmn/vim-minisnip),
[miniSnip](https://github.com/Jorengarenar/miniSnip), and the predecessor
[snipmate](https://github.com/garbas/vim-snipmate)?

**Comment by Konfekt** [comment]
From what I could quickly gather, the two 'mini' alternatives use a proper syntax not necessarily compatible to that used by snipmate.

**Comment by habamax** [comment]
Hi @Konfekt, yeah the name clashes with other plugins (I didn't check it back then to be frank). :)

I am not sure about other plugins based on snipMate, but my goal was/is to have something simple and "good enough" for me. I have used ultisnips and while it is a great snippet plugin it is too powerful for my needs. I tried snipmate and liked its simplicity. But then again after some usage I have found quirks here and there:

1. latin only support
2. interpolation implementation
3. some minor bugs
4. plugin internal structure
5. maybe some more

I would do a PR but original snipmate is not supported anymore.
